src/app.js

```diff
@@ -26,52 +26,55 @@ function main(sources) {
     time: new Date()
   }]);
 
+  const intent = DOM => {
+    const changeMessage$ = sources.DOM
+      .select('.input-message')
+      .events('input')
+      .map(e => e.target.value)
+      .share();
```

Comment:
Something like that, yeah!

Add `.do(x => console.log('foo'))` here, before `share`, and try it with and without `share`. You should see what effect it has :-)

`Observable.share` just means any consumers will share the same subscription, rather than creating their own.